js/memory-matching.js

```diff
@@ -1,19 +1,27 @@
-
-
 var matchingGame = {
-	elapsedTime: 0
+    elapsedTime: 0
+  , github: new GitHub()
 };
 
 matchingGame.deck = []
 
-// Like this https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/slice
-
-Hubbers["hubbers"].sort(shuffle).slice(0,8).map(function(hubber){
-	matchingGame.deck.push(hubber)
-	matchingGame.deck.push(hubber)
-});
+function getHubbers(callback) {
+   matchingGame.github.get("orgs/github/members", { all: true }, function (err, data) {
+        if (err) {
+            console.warn(err);
+            data = window.Hubbers;
+        }
+        callback(null, { hubbers: data });
+    });
+}
 
-console.log(matchingGame.deck)
+function shuffleHubbers(hubbers) {
+    // Like this https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/slice
+    hubbers.hubbers.sort(shuffle).slice(0,8).map(function(hubber){
```

Comment:
Oh interesting. Based on the link, it sounds like the Fisher-Yates shuffle algorithm would be the best/most correct implementation, yeah?